### DISCUSSION TITLE
Trying to open PR on uBlock repo as new contributor

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2939] [General] [answered]
@gorhill 

Can you please take a look at my PR? I tried to submit it to the uBlock repo, however I am blocked as a new contributor.

My PR fixes what I believe is a typo in a previous commit from you earlier today.

https://github.com/gorhill/uBlock/compare/master...jambohawk:uBlock:master

Thank you

**Comment by gorhill** [comment, marked as answer]
Oops, thanks for spotting and fixing this. I've pulled your commit into uBlock: <https://github.com/gorhill/uBlock/commit/971f36c78eda6fdaf7b1e857cbd9c2d2cc1bce36>

By doing so you are now contributor and you will now be able to submit PRs.

**Comment by ghost** [comment]
> Oops, thanks for spotting and fixing this. I've pulled your commit into uBlock: [gorhill/uBlock@971f36c](https://github.com/gorhill/uBlock/commit/971f36c78eda6fdaf7b1e857cbd9c2d2cc1bce36)

You're Welcome. :smiley: 

> By doing so you are now contributor and you will now be able to submit PRs.

Sounds good to me. Thank you.
